From c9828291218cd363f099ea90f09cc66cf13f86ba Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Thu, 3 Sep 2020 14:08:44 +0300
Subject: [PATCH 3/3] lx2160acex7: assert SUS_S5# GPIO to poweroff the COM

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 plat/nxp/soc-lx2160/aarch64/lx2160.S | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/plat/nxp/soc-lx2160/aarch64/lx2160.S b/plat/nxp/soc-lx2160/aarch64/lx2160.S
index 7b2b67b..83f5889 100644
--- a/plat/nxp/soc-lx2160/aarch64/lx2160.S
+++ b/plat/nxp/soc-lx2160/aarch64/lx2160.S
@@ -668,6 +668,10 @@ _soc_sys_off:
       * spurious timeout reset request
       * workaround: MASK RESET REQ RPTOE
       */
+    mov x3, #NXP_GPIO3_ADDR
+    mov w1, #0x01000000
+    str w1, [x3]
+
     ldr  x0, =NXP_RESET_ADDR
     ldr  w1, =RSTRQMR_RPTOE_MASK
     str  w1, [x0, #RST_RSTRQMR1_OFFSET]
-- 
2.17.1

