From 930d5e90e791dc357d271e1d71c78affe848625b Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Sun, 24 Jan 2021 13:05:46 +0200
Subject: [PATCH] plat: nxp: add tool print when choosing autodetect mode

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 plat/nxp/tools/create_pbl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plat/nxp/tools/create_pbl.c b/plat/nxp/tools/create_pbl.c
index 7ee085757..1b067823d 100644
--- a/plat/nxp/tools/create_pbl.c
+++ b/plat/nxp/tools/create_pbl.c
@@ -142,6 +142,7 @@ char *boot_src_string[] = {
 	"FLXSPI_NOR_BOOT",
 	"FLXSPI_NAND_BOOT",
 	"FLXSPI_NAND4K_BOOT",
+	"AUTODETECT"
 };
 
 enum stop_command {
-- 
2.25.1

