From 1aa6d2de8e68d37c39678bb6d21b9468ee1721bd Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Sun, 28 Nov 2021 13:33:10 +0200
Subject: [PATCH 3/8] lx2160acex7: assert SUS_S5# GPIO to poweroff the COM

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 plat/nxp/soc-lx2160a/aarch64/lx2160a.S | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/plat/nxp/soc-lx2160a/aarch64/lx2160a.S b/plat/nxp/soc-lx2160a/aarch64/lx2160a.S
index cc679f2ba..33c880509 100644
--- a/plat/nxp/soc-lx2160a/aarch64/lx2160a.S
+++ b/plat/nxp/soc-lx2160a/aarch64/lx2160a.S
@@ -562,6 +562,10 @@ endfunc _soc_sys_reset
  * uses x0, x1, x2, x3
  */
 func _soc_sys_off
+	/* assert CEX7 SUS_S5 / shutdown atx psu */
+	mov x3, #NXP_GPIO3_ADDR
+	mov w1, #0x01000000
+	str w1, [x3]
 
 	/* disable sec, QBman, spi and qspi */
 	ldr  x2, =NXP_DCFG_ADDR
-- 
2.35.3

