From f37a22f89c8c09f9f9d5dcbcf5dce431b955f574 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Fri, 8 Aug 2025 16:48:32 +0200
Subject: [PATCH 08/14] arch: arm: dts: fsl-lx2160a.dtsi: add pcs phys

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 arch/arm/dts/fsl-lx2160a.dtsi | 253 ++++++++++++++++++++++++++++++++++
 1 file changed, 253 insertions(+)

diff --git a/arch/arm/dts/fsl-lx2160a.dtsi b/arch/arm/dts/fsl-lx2160a.dtsi
index 680c69c7b73..4320a8d08bf 100644
--- a/arch/arm/dts/fsl-lx2160a.dtsi
+++ b/arch/arm/dts/fsl-lx2160a.dtsi
@@ -487,108 +487,126 @@
 			dpmac1: dpmac@1 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x1>;
+				pcs-handle = <&pcs1>;
 				status = "disabled";
 			};
 
 			dpmac2: dpmac@2 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x2>;
+				pcs-handle = <&pcs2>;
 				status = "disabled";
 			};
 
 			dpmac3: dpmac@3 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x3>;
+				pcs-handle = <&pcs3>;
 				status = "disabled";
 			};
 
 			dpmac4: dpmac@4 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x4>;
+				pcs-handle = <&pcs4>;
 				status = "disabled";
 			};
 
 			dpmac5: dpmac@5 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x5>;
+				pcs-handle = <&pcs5>;
 				status = "disabled";
 			};
 
 			dpmac6: dpmac@6 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x6>;
+				pcs-handle = <&pcs6>;
 				status = "disabled";
 			};
 
 			dpmac7: dpmac@7 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x7>;
+				pcs-handle = <&pcs7>;
 				status = "disabled";
 			};
 
 			dpmac8: dpmac@8 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x8>;
+				pcs-handle = <&pcs8>;
 				status = "disabled";
 			};
 
 			dpmac9: dpmac@9 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x9>;
+				pcs-handle = <&pcs9>;
 				status = "disabled";
 			};
 
 			dpmac10: dpmac@a {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0xa>;
+				pcs-handle = <&pcs10>;
 				status = "disabled";
 			};
 
 			dpmac11: dpmac@b {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0xb>;
+				pcs-handle = <&pcs11>;
 				status = "disabled";
 			};
 
 			dpmac12: dpmac@c {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0xc>;
+				pcs-handle = <&pcs12>;
 				status = "disabled";
 			};
 
 			dpmac13: dpmac@d {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0xd>;
+				pcs-handle = <&pcs13>;
 				status = "disabled";
 			};
 
 			dpmac14: dpmac@e {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0xe>;
+				pcs-handle = <&pcs14>;
 				status = "disabled";
 			};
 
 			dpmac15: dpmac@f {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0xf>;
+				pcs-handle = <&pcs15>;
 				status = "disabled";
 			};
 
 			dpmac16: dpmac@10 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x10>;
+				pcs-handle = <&pcs16>;
 				status = "disabled";
 			};
 
 			dpmac17: dpmac@11 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x11>;
+				pcs-handle = <&pcs17>;
 				status = "disabled";
 			};
 
 			dpmac18: dpmac@12 {
 				compatible = "fsl,qoriq-mc-dpmac";
 				reg = <0x12>;
+				pcs-handle = <&pcs18>;
 				status = "disabled";
 			};
 		};
@@ -613,6 +631,241 @@
 		#size-cells = <0>;
 		status = "disabled";
 	};
+
+	pcs_mdio1: mdio@8c07000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c07000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs1: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio2: mdio@8c0b000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c0b000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs2: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio3: mdio@8c0f000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c0f000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs3: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio4: mdio@8c13000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c13000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs4: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio5: mdio@8c17000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c17000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs5: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio6: mdio@8c1b000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c1b000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs6: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio7: mdio@8c1f000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c1f000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs7: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio8: mdio@8c23000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c23000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs8: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio9: mdio@8c27000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c27000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs9: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio10: mdio@8c2b000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c2b000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs10: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio11: mdio@8c2f000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c2f000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs11: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio12: mdio@8c33000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c33000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs12: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio13: mdio@8c37000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c37000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs13: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio14: mdio@8c3b000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c3b000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs14: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio15: mdio@8c3f000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c3f000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs15: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio16: mdio@8c43000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c43000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs16: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio17: mdio@8c47000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c47000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs17: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
+	pcs_mdio18: mdio@8c4b000 {
+		compatible = "fsl,fman-memac-mdio";
+		reg = <0x0 0x8c4b000 0x0 0x1000>;
+		little-endian;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+
+		pcs18: ethernet-phy@0 {
+			reg = <0>;
+		};
+	};
+
 	firmware {
 		optee {
 			compatible = "linaro,optee-tz";
-- 
2.43.0

