From 4d76da48691cf5d87fff2626065978617c629fbb Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Sun, 20 Dec 2020 16:06:26 +0200
Subject: [PATCH 28/28] lx2160acex7: add mtest start and end addresses

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 include/configs/lx2160acex7.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/lx2160acex7.h b/include/configs/lx2160acex7.h
index 9ba73fc0..d2ef78a2 100644
--- a/include/configs/lx2160acex7.h
+++ b/include/configs/lx2160acex7.h
@@ -77,5 +77,7 @@
 		"bootm $load_addr#$BOARD\0"
 
 #include <asm/fsl_secure_boot.h>
+#define CONFIG_SYS_MEMTEST_START		0x2080000000
+#define CONFIG_SYS_MEMTEST_END			0x2400000000
 
 #endif /* __LX2_CEX7_H */
-- 
2.25.1

